helpers/modalHelper.js

```diff
@@ -0,0 +1,11 @@
+const { setReminder } = require('./reminderHelper')
+
+async function replyToModal (interaction) {
```

Comment:
I would prefer HandleModal as a name here.
However this is simply a suggestion, and definately open for discussion